portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/rollingrestart/RollingRestartEE.testcase

```diff
@@ -1006,73 +995,6 @@ definition {
 		Portlet.viewPG(portletName = "Documents and Media");
 
 		User.logoutPG(specificURL = "http://localhost:9080");
```

Comment:
Before we only needed to logout because we have to login again after, but since this is our last step, there is no need to perform this step anymore. We don't really care about the actions of logging in / out to verify, since we are just verifying the login method in order to indirectly verify that the portal.property updated